ephios/settings.py

```diff
@@ -229,7 +229,7 @@
 SELECT2_I18N_PATH = ""
 
 # django-debug-toolbar
-if DEBUG:
+if DEBUG and env.bool("DEBUG_TOOLBAR", True):
```

Comment:
Disabling the toolbar while developing without needing to change the settings file.
I wanted one because it was making my dev server very slow while working on the templates.